include/grpc/impl/codegen/port_platform.h

```diff
@@ -114,6 +114,38 @@
 #define GPR_WIN32_ATOMIC 1
 #define GPR_MSVC_TLS 1
 #endif
+#elif defined(GPR_MANYLINUX1)
+// TODO(atash): manylinux1 is just another __linux__ but with ancient
+// libraries; it should be integrated with the `__linux__` definitions below.
```

Comment:
`manylinux1` is described as:
```
GLIBC <= 2.5
CXXABI <= 3.4.8
GLIBCXX <= 3.4.9
GCC <= 4.2.0
```
What's missing for me to Nike this is a map between all the feature `#define`s used by gRPC core and the versions of `glibc` and `libgcc`.